target/linux/bcm27xx/bcm2712/config-6.1

```diff
@@ -0,0 +1,600 @@
+CONFIG_64BIT=y
+# CONFIG_AIO is not set
+CONFIG_APERTURE_HELPERS=y
+CONFIG_ARCH_BCM=y
+CONFIG_ARCH_BCM2835=y
+CONFIG_ARCH_BINFMT_ELF_EXTRA_PHDRS=y
+CONFIG_ARCH_BRCMSTB=y
+CONFIG_ARCH_CORRECT_STACKTRACE_ON_KRETPROBE=y
+CONFIG_ARCH_DMA_ADDR_T_64BIT=y
+CONFIG_ARCH_HIBERNATION_POSSIBLE=y
+CONFIG_ARCH_KEEP_MEMBLOCK=y
+CONFIG_ARCH_MHP_MEMMAP_ON_MEMORY_ENABLE=y
+CONFIG_ARCH_MMAP_RND_BITS=18
+CONFIG_ARCH_MMAP_RND_BITS_MAX=24
+CONFIG_ARCH_MMAP_RND_BITS_MIN=18
+CONFIG_ARCH_MMAP_RND_COMPAT_BITS_MIN=11
+CONFIG_ARCH_NR_GPIO=0
+CONFIG_ARCH_PROC_KCORE_TEXT=y
+CONFIG_ARCH_SPARSEMEM_ENABLE=y
+CONFIG_ARCH_STACKWALK=y
+CONFIG_ARCH_SUSPEND_POSSIBLE=y
+CONFIG_ARCH_WANTS_NO_INSTR=y
+CONFIG_ARCH_WANTS_THP_SWAP=y
+CONFIG_ARM64=y
+CONFIG_ARM64_4K_PAGES=y
```

Comment:
@mj22226 according to your [previous comment](https://github.com/openwrt/openwrt/pull/13987#issuecomment-1854536319) I thought you were going to change this to 16K pages.
AFAIK, to reduce compatibility issues with certain applications we should go with 4K pages, but I don't think that there's any application used in OpenWrt that isn't compatible with 16K pages (but maybe I'm wrong). And the official Raspberry PI OS is using 16K pages for the RPi 5.
So I'm not really sure on what to do here :).
CC @graysky2

Reply:
@Noltari I found an issue with 16K pages with squashfs images, when installing multiple app or increasing rootfs size the rpi5 will not boot .

16K pages
```
[    4.942588] mount_root: loading kmods from internal overlay
[    4.954182] kmodloader: loading kernel modules from //etc/modules-boot.d/*
[    4.961263] kmodloader: done loading kernel modules from //etc/modules-boot.d/*
[    5.048328] block: attempting to load /tmp/overlay/upper/etc/config/fstab
[    5.055170] block: unable to load configuration (fstab: Entry not found)
[    5.061923] block: attempting to load /tmp/overlay/etc/config/fstab
[    5.068225] block: unable to load configuration (fstab: Entry not found)
[    5.074954] block: attempting to load /etc/config/fstab
[    5.083796] block: unable to load configuration (fstab: Entry not found)
[    5.090538] block: no usable configuration
[    5.094663] block: attempting to load /etc/config/fstab
[    5.099912] block: unable to load configuration (fstab: Entry not found)
[    5.106639] block: no usable configuration
[    5.110888] loop0: detected capacity change from 0 to 2129920
[    5.158056] loop0: detected capacity change from 2129920 to 2112768
[    5.175107] EXT4-fs (loop0): mounted filesystem with ordered data mode. Quota mode: disabled.
[    5.196700] block: attempting to load /tmp/overlay/upper/etc/config/fstab
[    5.204597] block: extroot: not configured
[    5.208733] block: attempting to load /etc/config/fstab
[    5.213984] block: unable to load configuration (fstab: Entry not found)
[    5.220731] block: no usable configuration
[    5.224981] mount_root: switching to ext4 overlay
[    5.230007] overlayfs: null uuid detected in lower fs '/', falling back to xino=off,index=off,nfs_export=off.
[    5.247626] insmod: module is already loaded - fat
[    5.252952] insmod: module is already loaded - vfat

[    5.046952] mount_root: overlay filesystem in /dev/loop0 has not been formatted yet
[    5.332227] EXT4-fs (loop0): mounted filesystem with ordered data mode. Quota mode: disabled.
[    5.340932] mount_root: overlay filesystem has not been fully initialized yet
[    5.348369] mount_root: switching to ext4 overlay
[    5.357425] overlayfs: null uuid detected in lower fs '/', falling back to xino=off,index=off,nfs_export=off.
[    5.373334] insmod: module is already loaded - fat
[    5.378604] insmod: module is already loaded - vfat
```
4K pages

```
[    6.629601] mount_root: loading kmods from internal overlay
[    6.642790] kmodloader: loading kernel modules from //etc/modules-boot.d/*
[    6.651185] kmodloader: done loading kernel modules from //etc/modules-boot.d/*
[    6.714975] block: attempting to load /tmp/overlay/upper/etc/config/fstab
[    6.721822] block: unable to load configuration (fstab: Entry not found)
[    6.728584] block: attempting to load /tmp/overlay/etc/config/fstab
[    6.734897] block: unable to load configuration (fstab: Entry not found)
[    6.741636] block: attempting to load /etc/config/fstab
[    6.748868] block: unable to load configuration (fstab: Entry not found)
[    6.755612] block: no usable configuration
[    6.759746] block: attempting to load /etc/config/fstab
[    6.765003] block: unable to load configuration (fstab: Entry not found)
[    6.771734] block: no usable configuration
[    6.775987] loop0: detected capacity change from 0 to 2129920
[    6.823466] loop0: detected capacity change from 2129920 to 1864832
[    6.839583] F2FS-fs (loop0): Mounted with checkpoint version = 726ec211
[    6.869192] block: attempting to load /tmp/overlay/upper/etc/config/fstab
[    6.877563] block: extroot: not configured
[    6.881702] block: attempting to load /etc/config/fstab
[    6.887029] block: unable to load configuration (fstab: Entry not found)
[    6.893784] block: no usable configuration
[    6.898574] mount_root: switching to f2fs overlay
[    6.904888] overlayfs: null uuid detected in lower fs '/', falling back to xino=off,index=off,nfs_export=off.
[    6.921569] insmod: module is already loaded - fat
[    6.926943] insmod: module is already loaded - vfat
```